doc/source/whatsnew/v1.1.0.rst

```diff
@@ -13,6 +13,32 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_110.astype_string:
+
+All dtypes can now be converted to ``StringDtype``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Previously, declaring or converting to :class:`StringDtype` was in general only possible if the data was already only ``str`` or nan-like.
+For example:
```

Comment:
can you use the 

Previous

Current

way of formatting (see other notes)

Reply:
I would personally just leave out the "previous". Saying that it didn't work before, and only showing the new feature seems sufficient to me.

Reply:
please use the prescribed format, its a standard that we have and consistency is key

Reply:
It's only a standard when there is actually a "Previous" to show. For bugs, we also don't show the error message you got before it was fixed. I personally see the fact that you couldn't do `astype("string")` to convert to strings as a bug (or not yet implemented feature, given how recent StringDtype is)